plugins/generic/pln/PLNPlugin.inc.php

```diff
@@ -329,6 +329,12 @@ function manage($verb, $args, &$message, &$messageParams) {
 				$this->import('classes.form.PLNSettingsForm');
 				$form = new PLNSettingsForm($this, $journal->getId());
 				
+				$application =& PKPApplication::getApplication();
+				$products =& $application->getEnabledProducts('plugins.generic');
+				if( ! isset($products['acron']) && Config::getVar('scheduled_tasks', 'Off') == 'Off') {
```

Comment:
Config::getVar will never return 'Off', I don't think. It should return PHP booleans when On/Off/True/False (case insensitive) are used.